vassal-app/src/main/resources/VASSAL/i18n/Editor.properties

```diff
@@ -386,6 +386,9 @@ Editor.ConfigureTree.add_cards=Add multiple cards
 Editor.ConfigureTree.add_pieces=Add multiple pieces
 # Adding a component (e.g. Add Map, Add Global Key Command, Add Zone, Add Single Piece)
 Editor.ConfigureTree.add_component=Add %1$s
+Editor.ConfigureTree.add_peer=Insert %1$s
+Editor.ConfigureTree.insert=Insert peer item
```

Comment:
I don't want no damn components peeing in my module! Can we make this either just 'Insert' or 'Insert component'? Peer is too technical.

And I don't think we need 'Insert ' in front of every sub-menu entry. That's implied by selecting Insert (component) from the menu.

Reply:
Done! (But who doesn't like to frolic with their peers? :D )